crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/storage.rs

```diff
@@ -69,8 +70,24 @@ pub fn calculate_variable_address(selector: Felt, key: Option<&[Felt]>) -> Felt
     }
 }
 
-fn storage_key(storage_address: Felt) -> Result<StorageKey, anyhow::Error> {
+#[must_use]
+pub fn variable_address(var_name: &str) -> Felt {
+    calculate_variable_address(felt_selector_from_name(var_name).into_(), None)
+}
+
+#[must_use]
+pub fn felt_selector_from_name(name: &str) -> EntryPointSelector {
```

Comment:
Nit

```suggestion
pub fn selector_from_name(name: &str) -> EntryPointSelector {
```

Reply:
This function was moved from other file, not created